src/test/java/BTC004Tests/Test1/Test1.java

```diff
@@ -0,0 +1,9 @@
+package BTC004Tests.Test1;
+
+public class Test1 {
+
+    // code1
+    // code2
```

Comment:
is this code necessary for this test automation? I guess need to run code3

Reply:
If u look at the ticket scenarios details u will see that need to run "code2"

Reply:
I agree wıth you @Ozleblebici